docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,184 @@
 <titleabbrev>Encrypting communications</titleabbrev>
 ++++
 
-{kib} supports Transport Layer Security (TLS/SSL) encryption for all forms of
-data-in-transit. Browsers send traffic to {kib} and {kib} sends traffic to {es}.
-These communications are configured separately.
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption for data-in-transit. While these terms are often used
+interchangeably, {kib} only supports TLS, which has superseded the old SSL protocols.
+
+Browsers send traffic to {kib} and {kib} sends traffic to {es}. These communications are configured separately to use TLS.
+
+TLS requires X.509 certificates to perform encryption and authentication of the application that is being communicated with. {kib} supports
+certificates and private keys in PEM or PKCS#12 format.
 
 [[configuring-tls-browser-kib]]
 ==== Encrypting traffic between the browser and {kib}
 
-NOTE: You do not need to enable {security-features} for this type of encryption. 
-
-. Obtain a server certificate and private key for {kib}.
+. *Obtain a server certificate and private key for {kib}.*
 +
 --
-{kib} supports certificates/keys in both PKCS #12 key stores and PEM format.
-
-When you obtain a certificate, you must do at least one of the following:
+{kib} will need to use this "server certificate" and corresponding private key when receiving connections from web browsers.
 
-.. Set the certificate's `subjectAltName` to the hostname, fully-qualified domain name (FQDN), or IP address of the {kib} server.
+When you obtain a server certificate, you must set its subject alternative name (SAN) correctly to ensure that modern web browsers with
+hostname verification will trust it. You can set the SAN to the {kib} server's fully-qualified domain name (FQDN), hostname, or IP address.
 
-.. Set the certificate's Common Name (CN) to the {kib} server's hostname or FQDN. Using the server's IP address as the CN does not work.
```

Comment:
I attempted to only use a CN (no SAN), and Chrome would not trust the certificate. I figured to simplify this section, we should just tell users to set a SAN for modern web browsers to trust the certificate correctly.